### PR TITLE
Change js object with classes

### PR DESCRIPTION
**Pull request by Said-laasri**
### _Awesome Books PR #3_
This PR is for adding all necessary files to the project and following estlint standards. This project is to keep a list of book titles and authors and store them in local storage. The usability of this project is adding books to the list or removing them and keeping tabs on your book list.
`index.html` for creating the form 
`bookList.js` Adding functions to `index.html` form
`style.css` for styling the web page
**Have fun 😊😉**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,3 +1,92 @@
 * {
   font-family: sans-serif;
 }
+
```
Comment (gemmen29) [inline style.css:6]: - [x] Kindly try to stick with the wireframe as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=Add%20CSS%20styles%20to%20the%20application%20to%20make%20it%20match%20this%20wireframe%3A)

| Your design | Wireframe | 
| --- | --- |
| image | image |

You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Said-laasri) [reply style.css:6]: @gemmen29 Thank you for reviewing my project, I will make the necessary corrections as suggested

**Review by algerina — APPROVED** [review]
Hi @Said-laasri , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed. :heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Said-laasri** [comment]
@algerina Thank you for reviewing my project.

**Pull request merged**